utilities/pegin-pegout-automation-demo/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "flyover-sdk-demo-cli",
+  "version": "1.0.0",
+  "description": "Interactive CLI demo for Flyover SDK authenticated pegin/pegout functionality",
+  "type": "module",
+  "main": "dist/index.js",
+  "bin": {
+    "flyover-demo": "dist/index.js"
+  },
+  "private": true,
+  "scripts": {
+    "build": "tsc",
+    "start": "node --disable-warning=DEP0040 dist/index.js",
```

Comment:
I think is ok to leave this warnings, otherwise we'll probably forget to fix them